### ISSUE TITLE
Where do you buy this?

### ISSUE THREAD

**Issue by immesys** [phishman/RDA1846#4]
This seems to be a useful IC, but I cannot find a distributor anywhere. Where did you source this IC from? (or its newer variant the AT1846s)?

**Comment by phishman** [comment]
It’s a Chinese chip, so the3 best source I’ve found and is fairly reliable is to use aliexpress.com<http://aliexpress.com>....  Delivery usually takes a couple weeks...

On Jun 2, 2018, at 2:26 PM, Michael Andersen <notifications@github.com<mailto:notifications@github.com>> wrote:


This seems to be a useful IC, but I cannot find a distributor anywhere. Where did you source this IC from? (or its newer variant the AT1846s)?

**Comment by iddq** [comment]
Have you built something with this chip?

**Comment by immesys** [comment]
Yeah, I built a few fox hunt beacons that can be detected with a cheap VHF/UHF handheld. But I ended up buying a module with it on. I didn't end up making a PCB from scratch because, for some reason, the modules are easy to get but the ICs themselves are harder

**Comment by immesys** [comment]
Something [like this](https://www.aliexpress.com/item/2043527225.html?spm=a2g0o.productlist.0.0.53ad3451Usgk8Q&algo_pvid=9c04185e-51d1-401d-a9ef-6572489d4add&algo_expid=9c04185e-51d1-401d-a9ef-6572489d4add-2&btsid=0ab6f83a15848928356522154e3029&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_), that uses the new 1846S chip

**Comment by iddq** [comment]
I don't understand why the two-way radios don't use the DTMF decoding capability of this chip...
